combat-trainer.lic

```diff
@@ -1171,6 +1183,13 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
+    @barb_buffs.each{|name| if Flags["ap-#{name}-expired"] then Flags.reset("ap-#{name}-expired"); start_barb_buff(name); end}
+  end
+
+  def start_barb_buff(name)
+    ability_data = get_data('spells').barb_abilities[name]
+    DRC.bput("#{ability_data['start_command']}", "#{ability_data['activated_message']}", 'But you are already', 'Your inner fire lacks')
```

Comment:
If I'm reading this right, one a buff drops due to insufficient IF, it'll never be reapplied. Is that correct?